dev-ruby/mime/mime-0.4.4.ebuild

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+USE_RUBY="ruby24 ruby25 ruby26 ruby27"
+
+inherit ruby-fakegem
+
+DESCRIPTION="A library for building RFC compliant MIME messages."
```

Comment:
We don't put `.` to DESCRIPTION,
https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Descriptive_text